mmv1/third_party/terraform/services/compute/resource_compute_region_instance_template.go.tmpl

```diff
@@ -1054,6 +1054,15 @@ be from 0 to 999,999,999 inclusive.`,
 					},
 				},
 			},
+
+			"key_revocation_action_type": {
```

Comment:
Could you add tests for this change as well?

Reply:
yup, will do